code/game/objects/items/devices/radio/encryptionkey.dm

```diff
@@ -1,159 +1,147 @@
-
-/obj/item/device/encryptionkey/
+/obj/item/device/encryptionkey
 	name = "standard encryption key"
 	desc = "An encryption key for a radio headset. Contains cypherkeys."
 	icon = 'icons/obj/radio.dmi'
 	icon_state = "cypherkey"
 	item_state = ""
-	w_class = 1
+	w_class = ITEMSIZE_TINY
 	slot_flags = SLOT_EARS
-	var/translate_binary = 0
-	var/translate_hive = 0
-	var/translate_hivenet = 0
-	var/syndie = 0
-	var/list/channels = list()
+	var/translate_binary = FALSE
+	var/translate_hive = FALSE
+	var/translate_hivenet = FALSE
+	var/syndie = FALSE // Signifies that it de-crypts Syndicate transmissions
+	var/list/channels = list("Common" = TRUE, "Entertainment" = TRUE)
 
-/obj/item/device/encryptionkey/attackby(obj/item/W as obj, mob/user as mob)
+/obj/item/device/encryptionkey/attackby(obj/item/W, mob/user)
+	return
 
 /obj/item/device/encryptionkey/syndicate
 	icon_state = "cypherkey"
-	channels = list("Mercenary" = 1)
+	channels = list("Common" = TRUE, "Entertainment" = TRUE, "Mercenary" = TRUE)
```

Comment:
You know, these could really be defines.

Reply:
![image](https://user-images.githubusercontent.com/22774890/77829565-80673000-712b-11ea-872c-7390a4dc6639.png)